pkg/assert/service.go

```diff
@@ -0,0 +1,15 @@
+package assert
+
+import (
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/gomega"
+)
+
+func CheckServiceSpecContainsSubstring(svc, expected string) {
+	cmd := "kubectl get svc "+svc+" -n default"+
+		" -o jsonpath='{range .items[*]}{.spec}' --kubeconfig="
+	res, err := shared.RunCommandHost(cmd + shared.KubeConfigFile)
```

Comment:
This is fine as it ends up being the same thing, but nit:
cmd should specify what the full command actually is, so we should add the `+ shared.KubeConfigFile` to the cmd itself instead of adding it when passing `cmd` to the other function.

Reply:
Thanks, updated